Hub documentation (#9090)
traefik/traefik: docs/content/traefik-hub/index.md

# Traefik Hub (Experimental)

## Overview

Once the Traefik Hub Experimental feature is enabled in Traefik,
Traefik and its local agent communicate together.

This agent can:

* get the Traefik metrics to display them in the Traefik Hub UI
* secure the Traefik routers
* provide ACME certificates to Traefik
* transfer requests from the SaaS Platform to Traefik (and then avoid the users to expose directly their infrastructure on the internet)

!!! warning "Traefik Hub Entrypoints"

>     When the Traefik Hub feature is enabled, Traefik exposes some services meant for the Traefik Hub Agent on a dedicated entrypoints (on ports `9900` and `9901` by default).

Reviewer: ```suggestion
    When the Traefik Hub feature is enabled, Traefik exposes some services meant for the Traefik Hub Agent on dedicated entrypoints (on ports `9900` and `9901` by default).
```

    Given their sensitive nature, those services should not be publicly exposed.

>     Also this dedicated entrypoint, regardless of how it is created (default, or user-defined), should not be used by anything other than the Hub Agent.

Reviewer: ```suggestion
    Also those dedicated entrypoints, regardless of how they are created (default, or user-defined), should not be used by anything other than the Hub Agent.
```
